### ISSUE TITLE
admin_logs embed content size error

### ISSUE THREAD

**Issue by Eccleria** [Titch88/ewibot#67]
Embed field value size is limited to 1024. Need to check for content length and divide it if necessary into multiple fields.

**Comment by Eccleria** [comment]
```javascript
    },
    files: []
  }
}
finishEmbed list error
 Message modifié 2022-08-31T17:11:13.615Z DiscordAPIError: Invalid Form Body
embeds[0].fields[3].value: Must be 1024 or fewer in length.
embeds[0].fields[4].value: Must be 1024 or fewer in length.
    at RequestHandler.execute (/home/ewibot/ewibot/node_modules/discord.js/src/rest/RequestHandler.js:350:13)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async RequestHandler.push (/home/ewibot/ewibot/node_modules/discord.js/src/rest/RequestHandler.js:51:14)
    at async ThreadChannel.send (/home/ewibot/ewibot/node_modules/discord.js/src/structures/interfaces/TextBasedChannel.js:176:15)
    at async finishEmbed (file:///home/ewibot/ewibot/src/admin/utils.js:97:23)
    at async Client.onMessageUpdate (file:///home/ewibot/ewibot/src/admin/listeners.js:617:23) {
  method: 'post',
  path: '/channels/1012365101594058762/messages',
  code: 50035,
  httpStatus: 400,
  requestData: {
    json: {
      content: undefined,
      tts: false,
      nonce: undefined,
      embeds: [Array],
      components: undefined,
      username: undefined,
      avatar_url: undefined,
      allowed_mentions: undefined,
      flags: undefined,
      message_reference: undefined,
      attachments: undefined,
      sticker_ids: undefined
    },
    files: []
  }
}
file:///home/ewibot/ewibot/src/admin/listeners.js:465
      addAdminLogs(msg.client.db, msg.id, "frequent", 6)
                              ^

TypeError: Cannot read properties of undefined (reading 'db')
    at file:///home/ewibot/ewibot/src/admin/listeners.js:465:31
    at Array.forEach (<anonymous>)
    at Client.onMessageDelete (file:///home/ewibot/ewibot/src/admin/listeners.js:464:17)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by Eccleria** [comment]
For messageDelete: Fixed in 1ade99c #79
